source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,310 @@
+===================
+Search Geospatially
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+On this page you will learn the how to search **geospatial data** in the
+MongoDB Java Driver.
+
+Geospatial data is data that represents a geographical location on
+the surface of the Earth. Examples of geospatial data include:
+
+- Locations of movie theaters
+- Borders of countries
+- Routes of bicycle rides
+- Dog exercise areas in New York City
+
+You should read this guide if you would like to:
+
+- Understand the different geospatial data formats in MongoDB.
+- Build geospatial queries.
+
+Coordinates on an Earth-like Sphere
+-----------------------------------
+
+You can represent geospatial data on an Earth-like sphere with **GeoJSON**.
+GeoJSON is a geospatial data format created by the Internet Engineering
+Task Force (IETF). You can use the GeoJSON format to store your data in MongoDB,
+and you must use the GeoJSON format to query your geospatial data in MongoDB. 
+
+For a concrete reference, GeoJSON data often resembles the following structure:
+
+.. code-block:: json
+
+   "<field name>" : {
+      "type": "<GeoJSON type>",
+      "coordinates": [
+         [<longitude>, <latitude>],
+         ...
+         [<longitude>, <latitude>]
+      ]
+   }
+
+For definitive information on GeoJSON, see the
+:rfc:`official IETF specification <7946>`.
+
+.. external resource
+
+GeoJSON Positions
+~~~~~~~~~~~~~~~~~
+
+A position is the base unit in GeoJSON, and represents a single place on Earth.
+A position is an array containing two or three number values: 
+
+- a longitude in the first position
+- a latitude in the second position
+- an optional elevation in the third position 
+
+.. tip:: Order Matters
+
+  GeoJSON orders coordinates as longitude first and latitude second. This may
+  be surprising as geographic coordinate system conventions generally list
+  latitude first and longitude second. Make sure to check what format any other
+  tools you are working with use. Popular tools such as OpenStreetMap and Google
+  Maps list coordinates as latitude first and longitude second. 
+  
+  Be careful when moving geospatial data between GeoJSON and other
+  formats, as you may need to swap your longitude and latitude values. 
+
+GeoJSON Types
+~~~~~~~~~~~~~
+
+Your GeoJSON object's type determines its geometric shape. Geometric shapes are
+made up of positions.
+
+Here are some common GeoJSON types and their corresponding positions:
+
+- ``Point``: a single position.
+- ``LineString``: a list of two or more positions, thus forming a series of line segments.
+- ``Polygon``: a list of positions in which the first and last
+  position are the same, thus enclosing some space.
```

Comment:
How about some simple examples here? An intersection or address might be a point, a road or the Great Wall of China might be a LineString, city boundaries or the Vatican might be delineated with a Polygon.

Reply:
Thanks for the note. I agree this could be helpful (and enjoyable) for learners. I like the examples of the The Great Wall of China and The Vatican. I used the Location of a Sculpture for point, and link to the wikipedia article for the sculpture which prominently features its location. 

Fix: